pkg/kube/client_test.go

```diff
@@ -119,10 +121,18 @@ func TestUpdate(t *testing.T) {
 	}
 
 	c := &Client{Factory: f}
-	if err := c.Update("test", objBody(codec, &listA), objBody(codec, &listB), false); err != nil {
+	if err := c.Update("test", objBody(codec, &listA), objBody(codec, &listB), false, 300, false); err != nil {
 		t.Fatal(err)
 	}
-
+	// //Test with a wait
+	// if err := c.Update("test", objBody(codec, &listB), objBody(codec, &listC), false, 300, true); err != nil {
+	// 	t.Fatal(err)
+	// }
+	// //Test with a wait should fail
```

Comment:
Question for these tests. These cause a panic because the mock client doesn't have a `ClientSet` method. Is there another way to test this?